appserver/website-bootstrap/main.js

```diff
@@ -160,12 +160,8 @@ function retrieveTrends(trendUrl) {
 }
 
 function getMoreInfo() {
-    let trend = this.innerHTML;
+    let trend = encodeURIComponent(this.innerHTML);
     let trend_news = null;
-    orig_trend= trend
-    if (trend.startsWith('#')){
-        trend = trend.substring(1);
```

Comment:
This might be a pointless oversight but, what if the '#' is in the middle of the trend string and is now present in the trend.substring(1) 
This should work fine though since I dont think there exist a twitter trend string that have a '#' in the middle

Reply:
the 'encodeURIComponent()' function deals with all non-ASCII characters everywhere in the given string

Reply:
the 'encodeURIComponent()' function deals with all non-ASCII characters everywhere in the given string